api/src/main/java/org/openmrs/module/locationbasedaccess/LocationBasedAccessConstants.java

```diff
@@ -13,4 +13,5 @@
 public class LocationBasedAccessConstants {
 
     public static final  String LOCATION_ATTRIBUTE_GLOBAL_PROPERTY_NAME = "locationbasedaccess.locationAttributeUuid";
+    public static final  String USER_PROPERTY_LOCATION_ATTRIBUTE_NAME = "locationUuid";
```

Comment:
Is the above constant name really correct?

Reply:
changed to "LOCATION_USER_PROPERTY_NAME"